tools/apidiff/Makefile

```diff
@@ -15,7 +15,7 @@ APIDIFF_DIR=.
 MONO_API_HTML_DIR = $(MONO_PATH)/mcs/tools/mono-api-html
 MONO_API_INFO_DIR = $(MONO_PATH)/mcs/tools/corcompare
 MONO_API_INFO = $(MONO_API_INFO_DIR)/bin/Debug/mono-api-info.exe
-MONO_API_HTML = $(MONO_API_HTML_DIR)/bin/Debug/mono-api-html.exe
+MONO_API_HTML = $(TOP)/external/macios-binaries/mono-api-html.exe
```

Comment:
I assume this will use the source code from mono once we use a branch that has the corresponding code?

Reply:
yes. I suspect there will be some adjustments in the short term and then it will become quite _static_ again. Backporting it (well all adjustments) to several release branches is time consuming.